core/embed/rust/src/ui/model_tr/component/button_controller.rs

```diff
@@ -40,7 +40,8 @@ enum ButtonState {
 
 pub enum ButtonControllerMsg {
     Pressed(ButtonPos),
-    Triggered(ButtonPos),
+    /// Which button was triggered, and whether it was pressed for a longer time
+    Triggered((ButtonPos, bool)),
```

Comment:
```suggestion
    Triggered(ButtonPos, bool),
```

Reply:
Done in [38a17f4](https://github.com/trezor/trezor-firmware/pull/3290/commits/38a17f47b34cb458ea6784b284da0d09192d61fc)